tools/logrotate_config/package_after_install_script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Create the log file for gcsfuse package.
+mkdir -p /var/log/gcsfuse/
+
+# Installs logrotate configuration files.
+mkdir -p /etc/logrotate.d/
+
+cat > /etc/logrotate.d/gcsfuse <<EOF
+/var/log/gcsfuse/*.log {
+    rotate ${LOGROTATE_FILES_MAX_COUNT:-5}
+    copytruncate
+    missingok
+    notifempty
+    compress
+    maxsize ${LOGROTATE_MAX_SIZE:-100M}
```

Comment:
Can we rename it to LOGROTATE_MAX_FILE_SIZE ?

Reply:
For now, we have fixed the value and user can change the config, we will provide documents for that.